rewrite-test/src/main/kotlin/org/openrewrite/java/cleanup/SimplifyBooleanReturnTest.kt

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.cleanup
+
+import org.junit.jupiter.api.Test
+import org.openrewrite.Recipe
+import org.openrewrite.RecipeTest
+import org.openrewrite.java.JavaParser
+
+interface SimplifyBooleanReturnTest : RecipeTest {
+
+    override val recipe: Recipe?
+        get() = SimplifyBooleanReturn()
+
+    @Test
+    fun simplifyBooleanReturn(jp: JavaParser) = assertChanged(
+        jp,
+        before = """
+                public class A {
+                    boolean ifNoElse() {
+                        if (isOddMillis()) {
+                            return true;
+                        }
+                        return false;
+                    }
+                    
+                    static boolean isOddMillis() {
+                        boolean even = System.currentTimeMillis() % 2 == 0;
+                        if (even == true) {
+                            return false;
+                        }
+                        else {
+                            return true;
+                        }
+                    }
+                }
+            """,
+        after = """
+                public class A {
+                    boolean ifNoElse() {
+                        return isOddMillis();
+                    }
+                    
+                    static boolean isOddMillis() {
+                        boolean even = System.currentTimeMillis() % 2 == 0;
+                        return !(even == true);
+                    }
+                }
+            """
+    )
+
+    @Test
+    fun dontSimplifyToReturnUnlessLastStatement(jp: JavaParser) = assertChanged(
+        jp,
+        before = """
+                public class A {
+                    public boolean absurdEquals(Object o) {
+                        if(this == o) {
+                            return true;
+                        }
+                        if(this == o) {
+                            return true;
+                        }
+                        return false;
+                    }
+                }
+            """,
```

Comment:
Pull this whole block left by one indentation line so that the """ is left aligned with `before`

Reply:
fixed; will keep in mind for future test formatting. feel free to unresolve if not right